### ISSUE TITLE
Two questions about the training and inference process

### ISSUE THREAD

**Issue by yuyu19970716** [antifen/mitosis-nuclei-detection#7]
Hello, I would like to ask, in the training of mmdet, the number of mitosis is very small, so will there be a category imbalance between mitosis and background, and the loss function needs to be modified to a weighted loss function?
     The  Second question: I have a lot of 20x data in my dataset, can I put it here and train together? For 20x data, I plan to trim the MMDET prediction to a size of 32*32 and do inference on MMCLASSIFICATION.
